spec/hashie/mash_spec.rb

```diff
@@ -1,5 +1,6 @@
 require 'spec_helper'
 
+# rubocop: disable Metrics/BlockLength
```

Comment:
Run `rubocop -a ; rubocop --auto-gen-config` instead.

Reply:
@dblock  I have problems making this to work. The above changes some files but does not make the build to pass (see my last commit.)